misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/core/MockAnnotationProcessor.java

```diff
@@ -320,33 +338,34 @@ private String formatPackageName(String value, BLangPackage parent) {
      *
      * @param functionName   Name of the function to mock
      * @param attachmentNode MockFunction object attachment node
+     * @return true if the provided function name valid
      */
-    private void validateFunctionName(String functionName, PackageID functionToMockID,
+    private boolean isValidFunctionName(String functionName, String moduleName, PackageID functionToMockID,
                                       AnnotationAttachmentNode attachmentNode) {
         if (functionToMockID == null) {
             diagnosticLog.logDiagnostic(DiagnosticSeverity.ERROR, attachmentNode.getPosition(),
-                    "could not find module specified ");
+                    "could not find specified module '" + moduleName + "'");
         } else {
             if (functionName == null) {
                 diagnosticLog.logDiagnostic(DiagnosticSeverity.ERROR, attachmentNode.getPosition(),
-                        "Function name cannot be empty");
+                        "function name cannot be empty");
             } else {
                 // Iterate through package map entries
                 for (Map.Entry<BPackageSymbol, SymbolEnv> entry : this.packageEnvironmentMap.entrySet()) {
                     if (entry.getKey().pkgID.equals(functionToMockID)) {
                         // Check if the current package has the function name
                         if (entry.getValue().scope.entries.containsKey(new Name(functionName))) {
                             // Exit validate function if the function exists in the entry
-                            return;
+                            return true;
                         }
                     }
                 }
                 // If it reaches this part, then the function has'nt been found in both packages
                 diagnosticLog.logDiagnostic(DiagnosticSeverity.ERROR, attachmentNode.getPosition(),
-                        "Function \'" + functionName + "\' cannot be found in the package \'"
-                                + functionToMockID.toString());
+                        "could not find function '" + functionName + "' in module '" + moduleName + "'");
```

Comment:
```suggestion
                        "cannot find the function '" + functionName + "' in module '" + moduleName + "'");
```

Reply:
What if the moduleName is not given by the user? Will it print the following? 
`cannot find the function func in module`

If the above happens, then lets reword it to remove the `in module` portion. The message would like:
`cannot find the function func`

Reply:
When the module name is not specified, it picks up the default module as the moduleName. Therefore the message is in the form `cannot find the function 'func' in module 'org/pkg_name:0.1.0'`.